src/content/ja/2019/caching.md

```diff
@@ -0,0 +1,691 @@
+
+以下のグラフは、上位10個の`Vary`ヘッダー値の人気を示しています。 `Accept-Encoding`はVaryの使用の90％を占め、`User-Agent`（11％）、`Origin`（9％）、`Accept`（3％）が残りの大部分を占めています。
+
+<figure>
+  <a href="/static/images/2019/16_Caching/fig14.png">
+    <img src="/static/images/2019/16_Caching/fig14.png" alt="図14.Varyヘッダーの使用率。" aria-labelledby="fig14-caption" aria-describedby="fig14-description" width="600" height="655" data-width="600" data-height="655" data-seamless data-frameborder="0" data-scrolling="no" data-iframe="https://docs.google.com/spreadsheets/d/e/2PACX-1vT3GWCs19Wq0mu0zgIlKRc8zcXgmVEk2xFHuzZACiWVtqOv8FO5gfHwBxa0mhU6O9TBY8ODdN4Zjd_O/pubchart?oid=384675253&amp;format=interactive">
+  </a>
+  <div id="fig14-description" class="visually-hidden">accept-encodingは90％で使用、ユーザーエージェントは10％-11％、オリジンは約7％-8％、acceptは少なく、cookie、x-forward-proto、accept-language、host、x-origin、access-control-request-method、およびaccess-control-request-headersの使用はほとんどありません</div>
+  <figcaption id="fig14-caption">図14.Varyヘッダーの使用率。</figcaption>
+</figure>
+
+## キャッシュ可能な応答にCookieを設定する
+
+応答がキャッシュされると、そのヘッダー全体もキャッシュにスワップされます。これが、DevToolsを介してキャッシュされた応答を検査するときに応答ヘッダーを表示できる理由です。
+
+<figure>
+  <a href="/static/images/2019/16_Caching/ch16_fig12_header_example_with_cookie.jpg">
+    <img src="/static/images/2019/16_Caching/ch16_fig12_header_example_with_cookie.jpg" alt="図15.キャッシュされたリソースのChrome開発ツール。" aria-labelledby="fig15-caption" aria-describedby="fig15-description" width="600" height="359">
+  </a>
+  <div id="fig15-description" class="visually-hidden">キャッシュされた応答のHTTP応答ヘッダーを示すChrome開発者ツールのスクリーンショット。</div>
+  <figcaption id="fig15-caption">図15.キャッシュされたリソースのChrome開発ツール。</figcaption>
+</figure>
+
+しかし、応答に`Set-Cookie`がある場合はどうなりますか？ [RFC 7234セクション8](https://tools.ietf.org/html/rfc7234#section-8)によると、`Set-Cookie`応答ヘッダーはキャッシングを禁止しません。これは、キャッシュされたエントリが`Set-Cookie`でキャッシュされている場合、それが含まれている可能性があることを意味します。 RFCでは、適切な`Cache-Control`ヘッダーを構成して、応答のキャッシュ方法を制御することを推奨しています。
+
+`Set-Cookie`を使用して応答をキャッシュすることのリスクの1つは、Cookieの値を保存し、後続の要求に提供できることです。 Cookieの目的によっては、心配な結果になる可能性があります。たとえば、ログインCookieまたはセッションCookieが共有キャッシュに存在する場合、そのCookieは別のクライアントによって再利用される可能性があります。これを回避する1つの方法は、`Cache-Control``プライベート`ディレクティブを使用することです。これにより、クライアントブラウザーによる応答のキャッシュのみが許可されます。
+
+キャッシュ可能な応答の3％に`Set-Cookieヘッダー`が含まれています。これらの応答のうち、`プライベート`ディレクティブを使用しているのは18％のみです。残りの82％には、パブリックおよびプライベートキャッシュサーバーでキャッシュできる`Set-Cookie`を含む530万のHTTP応答が含まれています。
+
+<figure>
+  <a href="/static/images/2019/16_Caching/ch16_fig16_cacheable_responses_set_cookie.jpg">
+    <img src="/static/images/2019/16_Caching/ch16_fig16_cacheable_responses_set_cookie.jpg" alt="図16. Set-Cookie応答のキャッシュ可能な応答。" aria-labelledby="fig16-caption" aria-describedby="fig16-description" width="600" height="567">
+  </a>
+  <div id="fig16-description" class="visually-hidden">応答の97％を示す棒グラフはSet-Cookieを使用せず、3％が使用します。この3％の内、15.3％がプライベート、84.7％がデスクトップ、モバイルは18.4％がパブリック、81.6％がプライベートであるという別の棒グラフに拡大されています。</div>
+  <figcaption id="fig16-caption">図16. <code>Set-Cookie</code>応答のキャッシュ可能な応答。</figcaption>
+</figure>
+
+## AppCacheおよびサービスワーカー
```

Comment:
Similar comment here. Both "AppCache" and "service workers" are technical terms so wouldn't translate either I don't think.

Reply:
I think line 614 same, I don't thinsk that this is translate.

Reply:
If we take MDN as an example they don’t translate service worker because it is an API (even if not an actual HTTP Header name): https://developer.mozilla.org/ja/docs/Web/API/Service_Worker_API

Reply:
Thanks, I understand and I agree.